### DISCUSSION TITLE
Cannot build for Windows with given instructions

### DISCUSSION THREAD

**Discussion by mrgehring** [merbanan/rtl_433#2533] [Q&A]
Trying to do a Windows build, I made it up to this point, trying to build rtl_sdr ( from https://github.com/steve-m/librtlsdr )

* go into the cmake/modules folder and open FindLibUSB.cmake with a text editor

This file no longer exists in the newer versions, looks like it was eliminated a few years ago.

Is there an update to the Windows build instructions, of even better a binary?

I am new to github, sorry if I am missing something obvious.

Thanks

**Comment by zuckschwerdt** [comment]
We have prebuilt binaries at https://github.com/merbanan/rtl_433/releases

**Comment by mrgehring** [comment]
Thank you very much for this reply! I have it working now. I did run into the problem with libwinpthread-1.dll , but I searched the repository and found a version posted that solves that problem.

I am grateful that you folks have written this code, it will help me to record my freezer / refrigerator temperatures so I can debug a problem.

**Comment by zuckschwerdt** [comment]
We had a problem of a broken build, seemingly missing libwinpthread, but that should be fixed. We don't use pthreads on Windows and it should not be needed to build or link.
